docs/templates.md

```diff
@@ -0,0 +1,26 @@
+# Decentralized Application Templates
+
+The templates can be found [here](https://github.com/scale-it/algo-builder-templates) and are extremely helpful and easy to use. They are designed and implemented in such a way that they can provide the aspiring developers a headstart in building dApps based on `Algorand Blockchain`. The templates heavily use [algosdk](https://github.com/algorand/js-algorand-sdk), [algo-builder](https://github.com/scale-it/algo-builder) and [AlgoSigner](https://github.com/PureStake/algosigner).
+
+## Usage
+
+An `algob` project and an `react` frontend are encapsulated in each template. Deployment information is stored in `checkpoints` (in `/artifacts`) from which user can import information directly (loading `.yaml` file in react-app).
+
+The templates can be easily imported or unboxed using the `algob unbox-template` command. 
+
+After successfully unboxing the template, please link the `algob` package in the template directory to use it for running scripts.
+
+
+## Quick Start
+
+The first step is to link `algob` package in the global bin. The documentation for the same can be found [here](https://github.com/scale-it/algo-builder/blob/master/packages/algob/README.md).
+
+`algob unbox-template <template-name> <destination-directory> --force (flag)`
+ - if `destination-directory` is not passed then current directory will be used.
+ - if `template-name` is not passed, then by default template "/default" is unboxed.
+ - if `--force` is passed, then files are overwritten. If it isn't passed, then user is made to choose whether to overwrite those files or not.
+ - if `template-name` passed is not present as an existing template, the command provides an interactive way to choose from the existing templates.
+ - The command also asks if the user wants to install the dependencies as a part of the current process.
+
+
```

Comment:
Can we also add an example-usage section for the `algob unbox-template` command ? (showing various use-cases of the command). Eg.
```
algob unbox-template default ./temp // unboxes "default" template to `cwd`/temp
algob unbox-template default // unboxes "default" to `cwd` as destination is not passed
algob unbox-template xyz ./dapp // since xyz template doesn't exist, user is asked to select a template from one of the existing ones (default/htlc)..
... and so on
```

Reply:
I have added these.